scripts/oneoff/rename-supplier-registered-country.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+"""Rename registration country from 'gb' to 'country:GB'
+
+We currently only have one country code in the database and that's for suppliers who are registered in the UK. It's
+'gb'. We're going start using the country picker as supplied by registers, and the country codes are stored in a
+different format. This script is to migrate our existing data to fall in line with the new country codes.
+
+See: https://trello.com/c/FeBysBI7
+
+Usage:
+    ./scripts/oneoff/rename-supplier-registered-country.py <stage> <api_token> [--dry-run]
+"""
+import sys
+sys.path.insert(0, '.')
+
+from docopt import docopt
+from dmapiclient import DataAPIClient
+from dmscripts.helpers.env_helpers import get_api_endpoint_from_stage
+
+OLD_COUNTRY = "gb"
+NEW_COUNTRY = "country:GB"
+
+def rename_country(client, dry_run):
+    counter = 0
+    for supplier in client.find_suppliers_iter():
+        if supplier.get('registrationCountry') == OLD_COUNTRY:
+            if not dry_run:
+                client.update_supplier(
```

Comment:
Depending on how risk-averse you're feeling we could do a backoff thing to re-try failed calls, something like this: https://github.com/alphagov/digitalmarketplace-scripts/blob/859f11f78124806c392f9f2ebd1f1e144f0c5caf/scripts/oneoff/migrate-supplier-data-from-declarations.py#L192

But as this can easily be re-run over again anyway maybe it's quicker to just try it and hope for the best.

Reply:
Have added, might come in useful!